services/src/main/java/org/keycloak/events/log/JBossLoggingEventListenerProviderFactory.java

```diff
@@ -88,6 +97,18 @@ public List<ProviderConfigProperty> getConfigMetadata() {
                 .options(logLevels)
                 .defaultValue("warn")
                 .add()
+                .property()
+                .name("sanitize")
+                .type("boolean")
+                .helpText("If true the log messages are sanitized to avoid line breaks. If false messages are not sanitized.")
+                .defaultValue("true")
```

Comment:
Should we mention this in release notes since it is changing default behavior?

Reply:
Yes, for sure, I wanted to add some notes in the upgrading notes. But I wanted to first share the PR to see if it was in the right direction for everybody.

Reply:
I have added the upgrading point explaining the changes. :smiley: